Mianta/Feb-15-Mianta/loops_challenge.py

```diff
@@ -0,0 +1,95 @@
+# # Print out the list after your loop to check if it got populated correctly.
+for i in range(len(days)):
+    print(f'On {days[i]}, favorite activity is to {activities[i]}.') #this is where I used the range function, and used (len) to list the days on seperate lines asking user what fav activity is 
+
+# 1.5: After your first loop, let's create a new one. As an example, let's say the user's favorite thing to
+# do on mondays is plan their week. This time, we want the output to be something like this:
+# 'On Mondays, your favorite activity is to plan your week.'
+# We need information from both lists! Let's use the `range` function to loop through the indexes
+# of the items in the lists (this will work because the lists are the same length).
+# Each time through this new loop, use the index number to index into each of your lists for the data
+# you need to print out.
+
+
+# Take a look back at the code you just wrote. Look at how much it does!
+# Often, programmers will be given large tasks, and it's our responsibility to be able to break it down into smaller pieces. We did the above piece by piece, but think about what the prompt might have been to get us there.
+
+# Maybe: Write a program that asks the user about their favorite thing to do each day of the week. Afterward, print out for the user each of their favorite daily activities.
+
+#Would this larger task have felt doable without breaking it down into steps? Is it clear what needs to be done?
+
+####### This defintely is easier to see what I need to do in order to get the loop to function properly. 
+
+
+# Try to break down the steps required for this second loop challenge.
+    # 1- Created a the day variable from the list of days
+
+    # 2- created a loop to take users input to determine what their favorite activity is on a given day.
+
+    # 3- saved user input and added to the activites variable. 
+
+    # 4- this is where I used the range function, and used (len) to list the days on seperate lines asking user what fav activity is 
+
+# QUESTION 2: For loops with list:
+
+# Write a program that loops through the days in the week. Each day, ask the user what the temperature is. 
+days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+
+temperature = '' #this is the list for integers
+
+for day in days:
+    temperature = int(input(f"What is the temperature {day}?"))#Wrote temperature = an integer then takes user numerical input to ask what temperature for each day
+
+print(temperature) #prints temp in a string
+    
+if temperature < 50:
+        print('Put on a jacket!')# If the temperature is below 50, tell the user to put on a jacket
+elif temperature >= 50 and temperature <= 65:
+        print('Put on a sweater!')#if the temperature is between 50 and 65, tell the user to put on a sweater.
+else:
+        print('Put on sunscreen!')#Finally, if the temperature is above 65, tell the user to put on some sunscreen.
+ 
```

Comment:
I think what the prompt really wants is to print out the clothings or sunscreen for each loop, so the if else statement should be inside the for day in days loop.

Reply:
made changes and doing a pull request now

Reply:
Hmmm. It's so weird. I don't see the changes you made. The files you pushed seem to be the same as the ones you pushed before.